ui/src/components/Header/Controller.tsx

```diff
@@ -84,10 +91,7 @@ export const Controller = (): ReactElement => {
     ddClient.docker.cli.exec('run', args, {
       stream: {
         onOutput(data): void {
-          if (data.stderr
-            && !data.stderr.includes('Successfully') // Api key activation is included in the error stream
-            && !data.stderr.includes('Execution of "prepare_host"')) {
-
+          if (!EXCLUDED_ERROR_TOAST.some(item => data.stderr.includes(item))) {
```

Comment:
nit: seems a bit easier to read if we assign the result of the function to a local variable (may be more of a personal taste, though). Something like:
```suggestion
          const isExcludedLog = EXCLUDED_ERROR_TOAST.some(item => data.stderr.includes(item));
          if (!isExcludedLog) {
```